rskj-core/src/test/java/co/rsk/net/LightProcessorTest.java

```diff
@@ -141,4 +142,48 @@ private static TransactionReceipt createReceipt() {
 
         return receipt;
     }
+
+    @Test
+    public void processTransactionIndexRequestMessageAndReturnsTransactionIndexCorrectly() {
```

Comment:
Move the test behind the last tests, and push the private methods to the bottom.